src/hazelweb/gui/Cell.re

```diff
@@ -65,7 +69,22 @@ let view =
               inject(a),
             ]);
           if (JSUtil.is_movement_key(evt)) {
```

Comment:
I think is_movement_key should probably return false for the ctrl+alt+arrow combos, since thats not a normal movement key